site/docs/contribute.md

```diff
@@ -422,6 +422,10 @@ Use `this` when assigning values to instance variables, making it clear when the
 2. Use `.` to create a hierarchy of config groups
     * For example, `s3` in `s3.access-key-id`, `s3.secret-access-key`
 
+#### Block Spacing
+
+To improve readability and maintain consistency, always place a newline after control blocks (if, for, while, switch, etc.). This helps separate logical sections of the code, making it easier to read and debug.      
+
```

Comment:
For new contributors, we keep on repeating this guideline. 
Documenting it under here can help reduce the effort.

Reply:
you might want to add a simple example to show what is meant here

Reply:
done